nullaway/src/main/java/com/uber/nullaway/handlers/InferredJARModelsHandler.java

```diff
@@ -79,8 +79,10 @@ public InferredJARModelsHandler() {
         parseStubStream(androidStubxIS, "android.jar: " + ANDROID_ASTUBX_LOCATION);
         LOG(DEBUG, "DEBUG", "Loaded Android RT models.");
       }
+    } catch (ClassNotFoundException e) {
+      LOG(DEBUG, "DEBUG", "Cannot find Android RT models locator class.");
```

Comment:
Nit: Make a documenting comment over this line and/or add to debug message something like "this is expected if the Android SDK JarInfer models Jar has not been set up for this build."